spellchecker.js

```diff
@@ -0,0 +1,21 @@
+module.exports = {
+  plugins: ["spellcheck"],
+  rules: {
+    "spellcheck/spell-checker": [
+      "warn",
+      {
+        comments: false,
+        ignoreRequire: true,
+        skipWords: [
+          'bg',
+          'cx',
+          'formatter',
+          'idx',
+          'num',
+          'postfix',
+          'svg',
+        ],
+      },
+    ],
```

Comment:
```suggestion
    "spellcheck/spell-checker": ["warn"],
```

이 레포가 유지보수가 활발한게 아니고, 어차피 대부분의 사용처에서 override할거라 굳이 안넣어도 될 것 같아요.

Reply:
@J132134 7/7 팀미팅에서 얘기 나눴던것 처럼
`comments` 와 `ignoreRequire` 만 남겨놓겠습니다.